hnn_core/tests/test_network.py

```diff
@@ -35,6 +42,17 @@ def test_network():
     # Assert that an empty Spikes object is created as an attribute
     assert net.spikes == Spikes()
 
+    # Assert that all external feeds are initialized
+    net._create_all_spike_sources()
+    n_evoked_sources = 270 * 3
+    n_pois_sources = 270
+    n_gaus_sources = 270
+    n_common_sources = 2
```

Comment:
are these values hardcoded for the network? Or does it depend on the params?

Reply:
they depend on the params